rsi/web.py

```diff
@@ -0,0 +1,53 @@
+"""
+Web helpers to download .dmi files as .rsi
+"""
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Optional
+from urllib.request import urlopen
+
+from rsi import Rsi
+
+
+def web_dmi_to_rsi(url: str) -> "Rsi":
+    """
+    Takes a web-url of a Dmi blob and returns an Rsi object
+    """
+    with TemporaryDirectory() as tmpdir_name:
```

Comment:
Why make a whole temporary directory here instead of a single temporary file?

Reply:
I've changed it to BytesIO now. I misread byondtools as the variable's called filename but it just gets passed to PIL so it can take it.